editor.planx.uk/src/@planx/components/Send/Editor.tsx

```diff
@@ -76,7 +84,8 @@ const SendComponent: React.FC<Props> = (props) => {
 
     // Show warnings on selection of BOPS or Uniform for likely unsupported services
     //   Don't actually restrict selection because flowSlug matching is imperfect for some valid test cases
-    const flowSlug = window.location.pathname?.split("/")?.[1];
+    const teamSlug = window.location.pathname?.split("/")?.[1];
+    const flowSlug = window.location.pathname?.split("/")?.[2];
```

Comment:
The `flowSlug` index was actually wrong here, but only impacted the in-editor warning messages - now fixed throughout!